hack/kiali/func-sm.sh

```
@@ -0,0 +1,435 @@
+  echo "${crd_list}" | ossm_confirm_and_delete_resource_lines "CRDs (*.istio.io, *.sailoperator.io, *.servicemesh*.io)"
+}
+
+delete_istio() {
+  infomsg "Deleting all Istio and IstioCNI CRs (if they exist) which uninstalls all the Service Mesh components"
+  local doomed_namespaces=""
+  for cr in \
+    $(${OC} get istio             -o custom-columns=K:.kind,N:.metadata.name,NS:.spec.namespace --no-headers | sed 's/  */:/g' ) \
+    $(${OC} get istiocni          -o custom-columns=K:.kind,N:.metadata.name,NS:.spec.namespace --no-headers | sed 's/  */:/g' ) \
+    $(${OC} get istiorevisiontags -o custom-columns=K:.kind,N:.metadata.name,NS:.spec.namespace --no-headers | sed 's/  */:/g' )
+  do
+    local res_kind=$(echo ${cr} | cut -d: -f1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)